cloud/jenkins/psmdb_operator_aks_version.groovy

```diff
@@ -1,28 +1,61 @@
 location='eastus'
 tests=[]
 clusters=[]
+release_versions="source/e2e-tests/release_versions"
 
 void verifyParams() {
-    if ("$RELEASE_RUN" == "YES" && (!"$PILLAR_VERSION" && !"$IMAGE_MONGOD")){
-        error("This is RELEASE_RUN. Either PILLAR_VERSION or IMAGE_MONGOD should be provided")
+    if ("$RELEASE_RUN" == "YES") {
+        echo "=========================[ RELEASE RUN ]========================="
+        if (!"$PILLAR_VERSION" && !"$IMAGE_MONGOD") {
+            error("Either PILLAR_VERSION or IMAGE_MONGOD should be provided for release run!")
+        }
     }
 }
 
-void getImage(String IMAGE_NAME) {
-    versions_file = "source/e2e-tests/release_images"
-    IMAGE = """${sh(
-        returnStdout: true,
-        script: "cat ${versions_file} | egrep \"${IMAGE_NAME}=\" | cut -d = -f 2 | tr -d \'\"\' "
-    ).trim()}"""
-    if ("$IMAGE") {
-        return "$IMAGE"
-    }
-    else {
-        error("Empty image is returned for $IMAGE_NAME. Check PILLAR_VERSION or content of file with images")
+String getParam(String PARAM_NAME) {
+    def param = "${params[PARAM_NAME]}"
+
+    if ("$param" && "$param" != "null" && param != "") {
```

Comment:
What if we use defaults? do not provide any params and do not have release build?

Reply:
Well, in that case this function will not be called whatsoever. Because we call it in the `if ("$RELEASE_RUN" == "YES") {}` block on line 46

Reply:
Right 🤦‍♀️

Reply:
Right 🤦‍♀️